sky/provision/kubernetes/utils.py

```diff
@@ -892,6 +892,53 @@ def check_credentials(context: Optional[str],
         return True, None
 
 
+def check_pod_config(cluster_yaml_path: str, dryrun: bool) \
```

Comment:
For reusability as a general method, we may want to use pod_config dict or V1Pod object as the arg
```suggestion
def check_pod_config(pod_config: Dict) 
```